From 0f21ba4dc6daf251eb6e58d4a8bbeb0395703685 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Wed, 29 May 2019 16:26:02 -0700
Subject: [PATCH 16/16] Remove RESOLVE_DEPENDENCY for VPU firmware

We'll provide VPU firmware via ARCHIVES, so no need to have this attempt
to download it at build time.
---
 inference-engine/cmake/dependencies.cmake | 18 ------------------
 1 file changed, 18 deletions(-)

diff --git a/inference-engine/cmake/dependencies.cmake b/inference-engine/cmake/dependencies.cmake
index 64435ca..217fbc9 100644
--- a/inference-engine/cmake/dependencies.cmake
+++ b/inference-engine/cmake/dependencies.cmake
@@ -37,24 +37,6 @@ else()
     set(MODELS_BRANCH "master")
 endif()
 
-if (ENABLE_MYRIAD)
-    RESOLVE_DEPENDENCY(VPU_FIRMWARE_MA2450
-            ARCHIVE_UNIFIED firmware_ma2450_491.zip
-            TARGET_PATH "${TEMP}/vpu/firmware/ma2450"
-            ENVIRONMENT "VPU_FIRMWARE_MA2450"
-            FOLDER)
-    debug_message(STATUS "ma2450=" ${VPU_FIRMWARE_MA2450})
-endif ()
-
-if (ENABLE_MYRIAD)
-    RESOLVE_DEPENDENCY(VPU_FIRMWARE_MA2480
-            ARCHIVE_UNIFIED firmware_ma2480_mdk_R7_9.zip
-            TARGET_PATH "${TEMP}/vpu/firmware/ma2480"
-            ENVIRONMENT "VPU_FIRMWARE_MA2480"
-            FOLDER)
-    debug_message(STATUS "ma2480=" ${VPU_FIRMWARE_MA2480})
-endif ()
-
 ## enable cblas_gemm from OpenBLAS package
 if (GEMM STREQUAL "OPENBLAS")
 if(NOT BLAS_LIBRARIES OR NOT BLAS_INCLUDE_DIRS)
-- 
2.21.0

